src/actors/common/cache.rs

```diff
@@ -207,39 +212,41 @@ impl<T: CacheItemRequest> Cacher<T> {
 
         let temp_file = tryf!(self.tempfile());
 
-        let future = request.compute(temp_file.path()).and_then(move |status| {
-            if let Some(ref cache_path) = cache_path {
-                sentry::configure_scope(|scope| {
-                    scope.set_extra(
-                        &format!("cache.{}.cache_path", name),
-                        cache_path.to_string_lossy().into(),
-                    );
-                });
+        let future = request
+            .compute(temp_file.path())
+            .and_then(move |status: CacheStatus| {
+                if let Some(ref cache_path) = cache_path {
+                    sentry::configure_scope(|scope| {
+                        scope.set_extra(
+                            &format!("cache.{}.cache_path", name),
+                            cache_path.to_string_lossy().into(),
+                        );
+                    });
+
+                    log::trace!("Creating {} at path {:?}", name, cache_path);
+                }
 
-                log::trace!("Creating {} at path {:?}", name, cache_path);
-            }
+                let byteview = ByteView::open(temp_file.path())?;
 
-            let byteview = ByteView::open(temp_file.path())?;
+                metric!(
+                    counter(&format!("caches.{}.file.write", name)) += 1,
+                    "status" => status.as_ref(),
+                );
+                metric!(
+                    time_raw(&format!("caches.{}.file.size", name)) = byteview.len() as u64,
+                    "hit" => "false"
+                );
 
-            metric!(
-                counter(&format!("caches.{}.file.write", name)) += 1,
-                "status" => status.as_ref(),
-            );
-            metric!(
-                time_raw(&format!("caches.{}.file.size", name)) = byteview.len() as u64,
-                "hit" => "false"
-            );
+                let path = match cache_path {
+                    Some(ref cache_path) => {
+                        status.persist_item(cache_path, temp_file)?;
+                        CachePath::Cached(cache_path.to_path_buf())
+                    }
+                    None => CachePath::Temp(temp_file.into_temp_path()),
+                };
 
-            let path = match cache_path {
-                Some(ref cache_path) => {
-                    status.persist_item(cache_path, temp_file)?;
-                    CachePath::Cached(cache_path.to_path_buf())
-                }
-                None => CachePath::Temp(temp_file.into_temp_path()),
-            };
-
-            Ok(request.load(key.scope.clone(), status, byteview, path))
-        });
+                Ok(request.load(key.scope.clone(), status, byteview, path))
+            });
```

Comment:
This code didn't change, I only added the type signature to the `|status: CacheStatus|` closure and the rest is rust-fmt's doing.